docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    depends_on:
+      - mongodb
+    build: .
+    restart: unless-stopped
+    environment:
+      # You can leave this as-is.
+      - DB_HOST=mongodb
+      - DB_USER=mongouser
+      - DB_PASSWORD=mongopasword
+      - DB_NAME=xyter
+      - DB_PORT=27017
+      # You have to set these values!
+      - DISCORD_TOKEN=YOUR_DISCORD_TOKEN
+      - DISCORD_CLIENT_ID=YOUR_CLIENT_ID
+      - DISCORD_DEV_GUILD_ID=YOUR_DEV_GUILD_ID
+      - HOSTER_NAME=YOUR_HOSTER_NAME
+      - NODE_ENV=development
+    stdin_open: true
+    tty: true
+  mongodb:
+    image: mongo:latest
+    restart: unless-stopped
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=mongouser
+      - MONGO_INITDB_ROOT_PASSWORD=mongopasword
+    ports:
+      - $MONGODB_LOCAL_PORT:27017
+    volumes:
+      - db:/data/db
```

Comment:
Could you make this as a ``bind mount``? I recommend keeping it outside of the bot code. So like ``root directory/mongodb`` or ```root directory/database`` or something in that way. Just so that the docker image isn't accessing it (app container).

Reply:
The volume don't access the app directory 😊 the ```db``` is a volume which is declared at the top it is saved at the local machine, the path ist something like ```/var/lib/docker/volumes```. And it saves the folder ```/data/db``` in that volume.

Reply:
I know but, I think many users will accidently remove their database if they updating Docker in a wrong way. I think it's better to do a bind mount, since they only need the database as a volume, this could be mounted for example to ``./database:/data/db`` or something by default